.github/workflows/build.yaml

```diff
@@ -0,0 +1,149 @@
+name: ci
+
+on:
```

Comment:
this file will still push the image, I think it will be easier to understand if this was "push" and what's left in the other file was "test_deploy"

Reply:
Renamed